### DISCUSSION TITLE
How do I pipe into  qsv stats?

### DISCUSSION THREAD

**Discussion by seven-of-eleven** [dathere/qsv#847] [Q&A] [answered]
How to pipe into qsv stats?

I have an air monitoring file with "Date,Time,Value,Unit,Device,Item" columns.
The following command:

qsv search -s "Date" 2023-03-05 joint-clean.sample.csv | qsv search -s "Device" AM-5110 | qsv search -s "Item" Methane | qsv stats | qsv table

returns nothing. If I remove "qsv stats" it returns the found data, but I'm looking for the stats summary of the found data.

[joint-clean.sample.csv](https://github.com/jqnatividad/qsv/files/10943404/joint-clean.sample.csv)

**Comment by jqnatividad** [comment, marked as answer]
@seven-of-eleven , this is regression I inadvertently introduced when I added the `sparsity` statistic. I need to do a rowcount to compute it, and doing so consumes stdin.

I'll need to create a tempfile when getting input from stdin for stats to fix this.

**Reply by seven-of-eleven** [reply]
Thanks for the response. I was scratching my head wondering what I was doing wrong. I'll work around it for now and revise my script when qsv is updated.

Thanks again. All the best.

**Reply by jqnatividad** [reply]
@seven-of-eleven and its been fixed - https://github.com/jqnatividad/qsv/pull/851...

It will be in the 0.92.0 release this weekend.

**Reply by jqnatividad** [reply]
@seven-of-eleven , just wanted to let you know 0.92.0 has been released. Please let me know if it resolves your issue.

**Reply by seven-of-eleven** [reply]
@jqnatividad Yes. Issue is resolved. Thank you!
